### DISCUSSION TITLE
flink doris  connector

### DISCUSSION THREAD

**Discussion by bdzxm** [apache/doris#9004] [A - General / Q&A]
doris  connector 是怎么与flink 做checkpoint配合的，似乎忽略了这一点吗

**Comment by bdzxm** [comment]
flink version 1.13.2

**Reply by JNSimba** [reply]
The previous flink connector did not provide semantic guarantees, and two-phase commit semantics will be added in the future, which can achieve Exactly once.
You can refer to https://github.com/apache/incubator-doris-flink-connector/pull/12

**Reply by bdzxm** [reply]
Thank you for you reply
